local_db/api.py

```diff
@@ -62,3 +72,13 @@ def get_requests_for_user(self, user: User):
             if user.output_checker or metadata.workspace in user.workspaces:
                 requests.append(self._request(metadata))
         return requests
+
+    def set_status(self, request: ReleaseRequest, status: Status, user: User):
+        with transaction.atomic():
+            # validate transition, permissions, and update object
+            super().check_status(request, status, user)
```

Comment:
Is there a reason to call `super().check_status` rather than `self.check_status`? I guess it doesn't make any difference unless the provider api does something extra with its own `check_status`, just wondering if it's deliberate and means subclasses _shouldn't_ override `check_status`

Reply:
Nope! C/p error, thanks, will fix.